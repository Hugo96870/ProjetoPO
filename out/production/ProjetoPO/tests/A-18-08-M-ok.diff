*** tests/A-18-08-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-18-08-M-ok.outhyp	2020-12-04 11:15:13.600010200 +0000
***************
*** 22,28 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 0|R2|C|7|210|189|10
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 22,28 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 0|R2|C|7|210|210|10
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
***************
*** 54,60 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|210|189
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
--- 54,60 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|210|0
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
