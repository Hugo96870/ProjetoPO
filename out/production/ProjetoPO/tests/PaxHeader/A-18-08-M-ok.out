30 mtime=1606841711.302325418
